src/rulesets/oas/functions/oasPathParam/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -33,11 +31,7 @@ exports[`oasPathParam Error if duplicate path parameters with same name are used
 Array [
   Object {
     "code": "path-params",
-    "message": "The path \\"**/foo/{bar}/{bar}**\\" uses the parameter \\"**{bar}**\\" multiple times.
-
-Path parameters must be unique.
-
-To fix, update the path so that all parameter names are unique.",
```

Comment:
The old message said they should be unique and the new one doesn’t. Can we bring that back?

Reply:
Sure Phil, thanks for spotting this, I'll create a PR to address that soon (certainly today, this is 21.05).